### DISCUSSION TITLE
exploring-the-dotnet-8-preview-the-minimal-api-aot-template/

### DISCUSSION THREAD

**Discussion by giscus** [andrewlock/blog-comments#382] [Announcements]
# exploring-the-dotnet-8-preview-the-minimal-api-aot-template/

In this post I discuss the Ahead of Time (AOT) compilation coming in .NET 8, look at it's benefits and limitations, and look at the new AOT template

https://andrewlock.net/exploring-the-dotnet-8-preview-the-minimal-api-aot-template/

**Comment by annakedrick** [comment]
On my website (https://learnaboutcat.com/), I have noticed two significant problems that require attention. Firstly, the website's loading times are noticeably slow, which can be frustrating for visitors. Secondly, some of the links on the website are unresponsive, making it difficult for users to navigate seamlessly. I have explored various solutions such as optimizing settings and prioritizing scripts, but none have produced the desired results thus far. I would appreciate guidance on how to effectively address these performance issues and ensure a smooth user experience on my website.

**Comment by Zenuka** [comment]
I think they renamed the template to [webapiaot](https://learn.microsoft.com/en-gb/aspnet/core/fundamentals/native-aot?view=aspnetcore-8.0#the-web-api-native-aot-template)
